tests/e2e/set_properties/CMakeLists.txt

```diff
@@ -6,3 +6,5 @@ copy_set_properties_e2e_python_files(common.py)
 copy_set_properties_e2e_python_files(set_properties.py)
 
 add_subdirectory(procedures)
+
+copy_e2e_files(set_properties workloads.yaml)
```

Comment:
Why are we copying workloads.yml suddenly?

Reply:
Need to handle running e2e tests based on what is built.

Before we would run all tests in the source test files, but `MG_EXPERIMENTAL_REPLICATION_MULTITENANCY` is required for these tests to pass. Hence now only workloads which should pass are copied to build directory, and we run based on them.